store/postgres/migrations/2021-05-21-165219_add_created_at_columns_for_deployment_schemas/up.sql

```diff
@@ -0,0 +1,5 @@
+alter table public.deployment_schemas
+    add column created_at timestamptz not null default now();
+
+alter table public.unused_deployments
+    add column created_at timestamptz not null;
```

Comment:
Since databases will have entries in `unused_deployments`, you'll need to populate the new column before setting it `not null`.  You'll have to do something like
```sql
alter table unused_deployments
        add column created_at timestamptz;
-- use a dummy date that is clearly made up
update unused_deployments set created_at = '2000-01-01';
alter table unused_deployments
        alter column created_at set not null;
```

Reply:
Got it.
It would be OK to use a placeholder value, like the one in your example?